src/operation.cpp

```diff
@@ -122,7 +135,16 @@ class OperationImpl : public Operation
         return std::visit([this](const auto& r) -> Feature::ValueType {
             using value_type = typename std::remove_reference_t<decltype(*r)>::value_type;
 
+            // Determine the type returned by the get() method in the Operation implementation
+            // class. The std::declval version of this doesn't work for in gcc 9
+            // (see https://github.com/isciences/exactextract/issues/82)
+#if __GNUC__ < 10
+            RasterStats<value_type> stats;
+            auto result = static_cast<const Derived*>(this)->get(stats);
+            using result_type = std::decay_t<decltype(result)>;
+#else
             using result_type = std::decay_t<decltype(static_cast<const Derived*>(this)->get(std::declval<RasterStats<value_type>>()))>;
+#endif
```

Comment:
@dbaston while I'm working on #84 over https://github.com/vincentsarago/exactextract/pull/1 I'm getting a compilation error 

```
        /project/src/operation.cpp:135:26:   required from ‘exactextract::Feature::ValueType exactextract::OperationImpl<Derived>::result_type() const [with Derived = exactextract::Quantile]’
        /project/src/operation.cpp:131:24:   required from here
        /project/src/operation.cpp:146:83: internal compiler error: in tsubst_copy, at cp/pt.c:16496
          146 |             using result_type = std::decay_t<decltype(static_cast<const Derived*>(this)->get(std::declval<RasterStats<value_type>>()))>;
              |                                
```

cibuildwheel use GCC 10 by default. I'll try to `fix` this but I'm absolutely not a C++ expert 😅

Reply:
changing `if __GNUC__ < 10` to `if __GNUC__ < 11` seems to fix the issue 🤷

Reply:
Fixed in [ce063d2](https://github.com/isciences/exactextract/commit/ce063d2ca9cb0e5139118a289a717fcd371c7b51), thanks.